packages/client/signers/src/communications/schemas.ts

```diff
@@ -132,7 +132,7 @@ export const ExportSignerPayloadSchema = {
                     .union([z.literal("ed25519"), z.literal("secp256k1")])
                     .describe("The cryptographic scheme to use"),
                 encoding: z
-                    .union([z.literal("base58"), z.literal("hex")])
+                    .union([z.literal("base58"), z.literal("hex"), z.literal("strkey")])
```

Comment:
Based on (this)[https://github.com/Crossmint/open-signer/pull/82#discussion_r2308971290] comment chain, maybe worth removing this now to prevent needing another SDK bump, but non-blocking!